doc/api/esm.md

```diff
@@ -341,15 +341,16 @@ will be used as the final fallback.
 
 The conditions supported in Node.js are matched in the following order:
 
-1. `"require"` - matched when the package is loaded via `require()`.
+1. `"node"` - matched for any Node.js environment. Can be a CommonJS or ES
+   module file. _This is currently only supported behind the
+  `--experimental-conditional-exports` flag._
+2. `"require"` - matched when the package is loaded via `require()`.
   _This is currently only supported behind the
   `--experimental-conditional-exports` flag._
-2. `"node"` - matched for any Node.js environment. Can be a CommonJS or ES
-   module file. _This is currently only supported behind the
+3. `"import"` - matched when the package is not loaded via `require()`.
```

Comment:
Should we make this a positive definition - "loaded via `import`" or at least mention that piece?

Reply:
Ok I've updated the text here to try to clarify the exact semantics more clearly. Let me know if that seems ok to you?